devel/stgit/Portfile

```diff
@@ -77,7 +71,7 @@ variant html_man_docs \
     lappend stgman ${worksrcpath}/Documentation/*.1
     depends_build-append    port:asciidoc port:xmlto
     post-build {
-        system -W ${worksrcpath} "PYTHON=${prefix}/bin/python2.7 make all doc"
+        system -W ${worksrcpath} "make all doc"
```

Comment:
Same here: the PYTHON environment variable is still needed. The `all` target has already been built by the post-build block in the main part of the portfile, so it does not need to be repeated here. (pre- and post- blocks are cumulative.) And there seems to be a lot to build so it takes a moment. It would build faster in parallel. So that we see the actual commands being run, turn on verbose mode. So all together, use `PYTHON=${python.bin} make doc -j${build.jobs} V=1`